virttest/qemu_vm.py

```diff
@@ -1174,14 +1174,14 @@ def add_rtc(devices):
             # Pay attention that rtc-td-hack is for early version
             # if "rtc " in help:
             if devices.has_option("rtc"):
-                cmd = " -rtc base=%s" % params.get("rtc_base", "utc")
-                cmd += _add_option("clock", params.get("rtc_clock", "host"))
-                cmd += _add_option("driftfix", params.get("rtc_drift", None))
-                return cmd
+                cmd = _add_option("base", params.get("rtc_base"))
+                cmd += _add_option("clock", params.get("rtc_clock"))
+                cmd += _add_option("driftfix", params.get("rtc_drift"))
```

Comment:
It is really nice to see many of these previously hard-coded options now configurable.